### PR TITLE
fix potential use after in tee_se_manager_unregister_reader()

### PR DESCRIPTION
**Pull request by viennadd**
Signed-off-by: Alex CHEN <viennadd@gmail.com>

### CODE REVIEW

**Comment by jforissier** [comment]
Thanks @viennadd.

Please provide a proper explanation of the issue and the fix in the commit log, and use your real name in the S-o-b: tag. We cannot accept pseudonyms (as specified in https://github.com/OP-TEE/optee_os/blob/master/Notice.md#contributions). The link to the issue should be added as a Link: tag.

File: core/tee/se/manager.c [hunk 1]
```diff
@@ -78,7 +78,7 @@ TEE_Result tee_se_manager_unregister_reader(struct tee_se_reader *r)
 	struct tee_se_reader_proxy *proxy;
 
 	mutex_lock(&ctx->mutex);
-	TAILQ_FOREACH(proxy, &ctx->reader_proxies, link)
+	TAILQ_FOREACH_SAFE(proxy, &ctx->reader_proxies, link)
```
Comment (jenswi-linaro) [inline core/tee/se/manager.c]: `TAILQ_FOREACH_SAFE()` expects four parameters.

**Comment by jforissier** [comment]
Commit log is not formatted properly (should be: subject line, one empty line, text, one empty line, tags; wrap at 75 characters max). I can fix that when merging. Anyway:
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by jforissier** [comment]
Merged manually (with cosmetic fixes to the commit message). Thanks @viennadd!

**Pull request closed without merging**